src/components/LoginForm/index.js

```diff
@@ -78,7 +73,15 @@ const LoginForm = ({ onSubmit }) => {
         onPress={handleSubmit}
         disabled={formHasErrors}
         style={styles.button}>
-        <Text style={styles.buttonTitle}>{status === LOADING ? LOADING_TEXT : SIGN_IN_TEXT}</Text>
+        {status === LOADING ? (
```

Comment:
FYI: I need two different id for testing so, I have to separate in two different Text component.